src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
I believe we shouldn't use since it's specifically the line height for `.expanding-table td` and it's probably higher height than the `25px` but if there is enough people asking to use that base after design agreement I'm fine switching.